index.html

```diff
@@ -33,6 +134,21 @@ <h3>LET’S CONNECT</h3>
       <li> <a href="https://www.hackerrank.com/manelhammouche3" target="_blank" rel="noopener"
           class="social_media_link"><i class="fa-brands fa-hackerrank fa-2x"></i> </a></li>
     </ul>
+    <button class="enabled_btn">Get my resume</button>
+    <ul id="skills">
+      <li class="skill">
+        <span>Languages</span>
+        <span><i class="fa-solid fa-chevron-down"></i></span>
+      </li>
+      <li class="skill">
+        <span>Frameworks</span>
+        <span><i class="fa-solid fa-chevron-down"></i></span>
+      </li>
+      <li class="skill">
+        <span>Skills</span>
+        <span><i class="fa-solid fa-chevron-down"></i></span>
+      </li>
```

Comment:
- [ ] [optiona] Your design is missing the languages section from the template, Please make sure you add them and it matches the design on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Your design

![image](https://user-images.githubusercontent.com/83803414/158964025-fadd7933-2f32-4c1f-96d8-1e23f3a458c0.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/158964077-9c0aa001-5b2a-48ef-93f9-7e48c72aec03.png)

Reply:
Hello @thecodechaser, thank you for your review. I didn't implement the "languages" part based on the project requirements, My thought was to fully implement it once we study JS. 
![image](https://user-images.githubusercontent.com/50721479/158964985-45994679-7e42-4be2-b812-6dad051a0d23.png)